src/app/Main/Body/QueryPage/Services/ContainerServices/fetchInitialData.service.tsx

```diff
@@ -11,11 +14,12 @@ import { generateUniqueValuesService } from "../generateUniqueValues.service";
 import { FilterConfigDTO } from "../../../../../Shared/Model/Api_Filter.model";
 
 export async function fetchInitialDataService(): Promise<{
-    filterStatus: number;
-    isolateStatus: number;
-    subFilters?: ClientSingleFilterConfig[];
-    uniqueDataValues?: FilterInterface;
-    totalNrOfIsolates?: number;
+    status: { filterStatus: number; isolateStatus: number };

```

Comment:
Actually i meant to use ApiResponse<...> as return type. But then you have to merge filterStatus and isolateStatus (i missed that), another refactoring... But the logic is the same now, so it is done.
This recommendation was meant for the fetchConditionalFiltersService too. Here you could merge the statusList in the service and just return one status.